framework/src/base/Assembly.C

```diff
@@ -665,6 +667,9 @@ Assembly::reinitFE(const Elem * elem)
     }
   }
 
+  for (unsigned int i = 0; i < _extra_elem_ids.size(); ++i)
```

Comment:
have to make a copy here because libMesh currently does not provide an interface function to return the reference of these extra integers partly because they are within a larger array.